backend/src/main/java/net/finman/dao/InvoiceDaoImpl.java

```diff
@@ -65,19 +73,18 @@ public void createInvoice(Invoice inv) throws ResourceNotCreatedException {
     private String generateOcr(){
         long ocr;
         SplittableRandom rng = new SplittableRandom();
-        ocr = rng.longs(1, 1000000000, 9999999999L).sum();  
+        ocr = rng.longs(1, 1000000000, 9999999999L).sum();
         return Long.toString(ocr);
     }
 
     @Override
-    public void addInvoiceItems(UUID serialNumber, String seller, List<Item> items) throws ResourceNotCreatedException {
+    public void addInvoiceItems(UUID serialNumber, String seller, List<InvoiceItem> items) throws ResourceNotCreatedException {
         try {
-            for (Item i : items) {
+            for (InvoiceItem i : items) {
                 SqlParameterSource itemsParams = new MapSqlParameterSource()
                         .addValue("invoice", serialNumber)
                         .addValue("seller", seller)
                         .addValue("name", i.getName())
-                        .addValue("item_owner", i.getOwner())
                         .addValue("amount", i.getAmount());
                 template.update(INSERT_INVOICE_ITEMS, itemsParams);
             }
```

Comment:
We don't need this function `addInvoiceItems(...)` anymore since invoiceItems are added through an Invoice directly